Time Tracker/Log.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+using nsButton;
+using nsCSV;
+
+namespace nsLog
+{
+    class Log
+    {
+        public Log(Stopwatch stopwatch)
+        {
+            this._stopwatch = stopwatch;
```

Comment:
Should a Log have a stopwatch? A log just writes something somewhere but... Stopwatch??